sdk/digitaltwins/digitaltwins/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@azure/digitaltwins",
+  "version": "1.0.0",
+  "description": "Azure NodeJs client library for Azure Digital Twins",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "dist/digitalTwinsClient.js",
+  "module": "dist-esm/index.js",
+  "types": "types/digitalTwinsClient.d.ts",
+  "scripts": {
+    "clean": "rimraf node_modules dist dist-esm test-dist temp types *.tgz *.log",
+    "prepack": "npm install && npm run build",
+    "pack": "npm pack 2>&1",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "generate-pl": "autorest --typescript --typescript.azure-arm=true --use=@microsoft.azure/autorest.typescript@5.1.0 --add-credentials --model-enum-as-union --license-header=MICROSOFT_MIT_NO_VERSION --output-folder=./src/generated --input-file=swagger/digitaltwins.json"
```

Comment:
do we need `--typescript.azure-arm=true` ?

Reply:
Yes we do. The paginated API generation is missing if we don't specify "arm" here. (Chris Radek figured it out)

---
In reply to: [447264176](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447264176) [](ancestors = 447264176)